mappings/net/minecraft/world/SpawnHelper.mapping

```diff
@@ -123,14 +133,21 @@ CLASS net/minecraft/unmapped/C_rvfygodd net/minecraft/world/SpawnHelper
 			ARG 3 chunkConsumer
 	CLASS C_tsgfqxij Info
 		FIELD f_abudamhp groupToCount Lit/unimi/dsi/fastutil/objects/Object2IntOpenHashMap;
+		FIELD f_epcjsxyx spawnDensityCapper Lnet/minecraft/unmapped/C_glzmmzpj;
 		FIELD f_fiqoycvb densityField Lnet/minecraft/unmapped/C_fyusbfrv;
 		FIELD f_khcrfzom spawningChunkCount I
 		FIELD f_oaklktzk cachedPos Lnet/minecraft/unmapped/C_hynzadkk;
 		FIELD f_strcujzm cachedDensityMass D
 		FIELD f_vmurapeq cachedEntityType Lnet/minecraft/unmapped/C_ogavsvbr;
 		FIELD f_zcovpoqd groupToCountView Lit/unimi/dsi/fastutil/objects/Object2IntMap;
+		METHOD <init> (ILit/unimi/dsi/fastutil/objects/Object2IntOpenHashMap;Lnet/minecraft/unmapped/C_fyusbfrv;Lnet/minecraft/unmapped/C_glzmmzpj;)V
+			ARG 1 spawningChunkCount
+			ARG 2 object2IntOpenHashMap
```

Comment:
Is this supposed to be here?

Reply:
wdum?

Reply:
The name is the default one instead of a proper name